### DISCUSSION TITLE
Stability AI Releases StableStudio, the Open-Source Future of DreamStudio

### DISCUSSION THREAD

**Discussion by cibernicola** [vladmandic/sdnext#1030] [General]
`From enabling local-first development to experimenting with a new[ plugin system](https://github.com/Stability-AI/StableStudio/blob/main/packages/stablestudio/README), we’ve tried hard to make things extensible for external developers. In addition to moving our active development into the public repository, we plan to create bounties for improvements and new features. Here are a few things we’d be excited to support…

Local inference through[ WebGPU](https://developer.mozilla.org/en-US/docs/Web/API/WebGPU_API)

Local inference through[ stable-diffusion-webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui)

Desktop installation

ControlNet tools`

https://stability.ai/blog/stablestudio-open-source-community-driven-future-dreamstudio-release

**Comment by vladmandic** [comment]
that would be so COOL to have as an alternative simple front-end.
and given the plugin system to have customizable backends, it should be more than doable!
i'm converting this to a feature request so it doesn't fall of the radar.

**Reply by vladmandic** [reply]
link to #1031
